internal/cmd/api-key/command_list.go

```diff
@@ -52,13 +52,10 @@ func (c *command) newListCommand() *cobra.Command {
 func (c *command) list(cmd *cobra.Command, _ []string) error {
 	c.setKeyStoreIfNil()
 
-	resourceType, clusterId, currentKey, err := c.resolveResourceId(cmd, c.V2Client)
+	clusterId, currentKey, err := c.resolveResourceId(cmd, c.V2Client)
 	if err != nil {
 		return err
 	}
-	if resourceType == resource.Cloud {
-		clusterId = resource.Cloud
-	}
```

Comment:
It looks like this change will cause clusterId to be empty since `resolveResourceId` still breaks out of the switch in the Cloud case without setting the clusterId; does this affect the `c.V2Client.ListApiKeys` call further down in this file?